### PR TITLE
Development footer

### PR DESCRIPTION
**Pull request by ehab-hassan**
### Description

Create footer for dashboard

### Changes

Added footer to dashboard
I removed the transition because it shows a footer at the top of the content.

### Related Issues

#1515

image

image

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by mohamedamer453 — COMMENTED** [review]
- The `mainFooter` component should be renamed to `main_footer` to be consistent with other components. 

- In some pages like `Virtual Machines`, `Orchestrators`, `Resource calculator`, etc ... there's a space that appears under the footer.
   image

**Comment by mohamedamer453** [comment]
> @mohamedamer453 can u hard refresh and check it works fine with me

After hard reload, the space is still there.

image

after further checking i found that it's related to the resolution that i'm using, i'm currently using a `2560 * 1440` monitor, but when i used a `1920 * 1080` it worked fine and the space wasn't there. so i think it should handle higher resolutions in a better way.

**Comment by ehab-hassan** [comment]
fixed
image

**Comment by mohamedamer453** [comment]
> fixed image

Indeed, the extra spaces at the bottom of the footer are now gone.

File: packages/playground/src/App.vue [hunk 1]
```diff
@@ -411,6 +419,7 @@ import AppTheme from "./components/app_theme.vue";
 import DeploymentListManager from "./components/deployment_list_manager.vue";
 import DisclaimerToolbar from "./components/disclaimer_toolbar.vue";
 import FundsCard from "./components/funds_card.vue";
+import main_footer from "./components/main_footer.vue";
```
Comment (mohamedamer453) [inline packages/playground/src/App.vue:422]:
```suggestion
import MainFooter from "./components/main_footer.vue";
```

File: packages/playground/src/App.vue [hunk 2]
```diff
@@ -181,6 +188,7 @@
           </v-container>
         </DeploymentListManager>
         <TFNotification v-if="hasActiveProfile && hasGrid" />
+        <main_footer />
```
Comment (mohamedamer453) [inline packages/playground/src/App.vue:191]:
```suggestion
        <MainFooter />
```

File: packages/playground/src/App.vue [hunk 3]
```diff
@@ -449,7 +458,11 @@ export default {
     ProfileManagerController,
     TfNavigationLoader,
     TfOfflineNotifier,
+    main_footer,
```
Comment (mohamedamer453) [inline packages/playground/src/App.vue:461]:
```suggestion
    MainFooter,
```

**Review by mohamedamer453 — COMMENTED** [review]
The name of the component file should be `main_footer` but the calls and the import should  be `MainFooter`.

**Review by mohamedamer453 — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**